packages/libraries/core/tests/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "include": ["."],
   "compilerOptions": {
     "baseUrl": ".",
-    "rootDir": ".",
+    "rootDir": "..",
```

Comment:
This is to make the local IDE happy since within the tests we are importing from the `../src`, and Typescript complains when importing files outside the root directory.